server/src/test/java/org/opensearch/bootstrap/BootstrapChecksTests.java

```diff
@@ -774,4 +779,22 @@ Version getVersion() {
         version.set(Runtime.version());
         BootstrapChecks.check(emptyContext, true, Collections.singletonList(check));
     }
+
+    public void testMultipleDataPathCheck() throws NodeValidationException {
```

Comment:
I think there should be 3 cases here:
1. search node, 1 data path
1. search node, 2 data paths
1. data node, 2 data paths
 
You have case 2 covered, but not really the other two. They should be separate test methods as well because it is generally a best practice to make each test case test one thing whenever possible.

Reply:
Got it!